torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -431,667 +103,6 @@ struct TLSCurrentInterpreterGuard {
-        if (auto class_type = node->inputs().at(0)->type()->cast<ClassType>()) {
-          emitCall(&class_type->getMethod(node->s(attr::name)), node->inputs());
-        } else {
-          emitInterfaceCall(node->s(attr::name), node->inputs());
-        }
-        break;
-      case prim::TypeCheck:
-        emitTypeCheck(node);
-        break;
-      case prim::BailOut:
-        emitBailOut(node);
-        break;
-      case prim::profile_ivalue:
-      case prim::profile:
-        emitProfile(node);
-        break;
-      case prim::GetAttr:
-        emitGetAttr(node);
-        break;
-      case prim::SetAttr:
-        emitSetAttr(node);
-        break;
-      case prim::ListUnpack:
-        emitListUnpack(node);
-        break;
-      case prim::TupleConstruct:
-        emitTupleConstruct(node);
-        break;
-      case prim::ListConstruct:
-        emitContainerConstruct(LIST_CONSTRUCT, node);
-        break;
-      case prim::DictConstruct:
-        emitContainerConstruct(DICT_CONSTRUCT, node);
-        break;
-      case prim::CreateObject:
-        emitCreateObject(node);
-        break;
-      case prim::isinstance:
-        emitIsinstance(node);
-        break;
-      case prim::TupleSlice:
-        emitTupleSlice(node);
-        break;
-      case prim::fork:
-        emitFork(node);
-        break;
-      case aten::warn:
-        emitWarn(node);
-        break;
-      case prim::Enter:
-        emitEnter(node);
-        break;
-      case prim::Exit:
-        emitExit(node);
-        break;
-    }
-  }
-
-  void emitCodeForBlock(Block* block) {
-    emitNodeAtBlockLevel(block->param_node());
-    for (auto node : block->nodes()) {
-      emitNodeAtBlockLevel(node);
-    }
-    emitNodeAtBlockLevel(block->return_node());
-  }
-
-  const std::vector<GraphExecutor*>& grad_executors() {
-    if (!grad_executors_) {
-      grad_executors_.emplace();
-      for (Operation& op : operator_table_) {
-        if (auto executor = detail::getGradExecutor(op)) {
-          grad_executors_->push_back(executor);
-        }
-      }
-    }
-    return *grad_executors_;
-  }
-
-  const std::vector<GraphExecutor*>& diff_graph_op_executors() {
-    if (!forward_executors_) {
-      forward_executors_.emplace();
-      for (Operation& op : operator_table_) {
-        if (auto executor = detail::getDifferentiableGraphOpExecutor(op)) {
-          forward_executors_->push_back(executor);
-        }
-      }
-    }
-    return *forward_executors_;
-  }
-
-  void dump(std::ostream& out, size_t i) const {
-    out << i << " " << instructions_[i];
-    if (instructions_[i].op == OP || instructions_[i].op == CALL ||
-        instructions_[i].op == OPN) {
-      out << " # " << *instructions_source_[i];
-    } else {
-      out << "\n";
-    }
-  }
-
-  void dump(std::ostream& out) const {
-    out << *graph_ << "\n";
-    for (size_t i = 0; i < instructions_.size(); ++i) {
-      dump(out, i);
-    }
-  }
-};
-
 struct MobileCodeImpl : CodeImpl {
```

Comment:
Could we move `MobileCodeImpl` to `code_impl.{h, cpp}` as well?

Reply:
sure.